Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -6911,23 +6911,15 @@ class BuildPlanTestCase: BuildSystemProviderTestCase {
     }
 }
 
-class BuildPlanNativeTests: BuildPlanTestCase {
-    override open var buildSystemProvider: BuildSystemProvider.Kind {
-        return .native
-    }
-
-    override func testDuplicateProductNamesWithNonDefaultLibsThrowError() async throws {
-        try await super.testDuplicateProductNamesWithNonDefaultLibsThrowError()
+final class BuildPlanNativeTests: BuildPlanTestCase {
```

Comment:
Marked all these subclasses `final` to make it clear we don't further extend such tests anywhere.